conda-envs/environment-dev.yml

```diff
@@ -30,6 +30,8 @@ dependencies:
 - pydata-sphinx-theme
 - pytest-cov>=2.5
 - pytest>=3.0
+- python-graphviz
+- scipy>=1.4.1
 - sphinx-copybutton
```

Comment:
We can also update the file and move sphinx-design to the conda section here instead of the pip one below. It is already on conda forge: https://anaconda.org/conda-forge/sphinx-design